apps/frontend/src/assets/css/main.css

```diff
@@ -0,0 +1,4 @@
+main {
```

Comment:
Remove these styles, we will have a separate layout with needed container. Or if we will need to adjust main styles, we will do this in scaffolding